### PR TITLE
json with marshaller + mockito testing

### PR DESCRIPTION
**Pull request by Taniann**

### CODE REVIEW

File: src/test/java/ua/mateacademy/tania/lesson11/StringChangerServiceMockSpyTest.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    @InjectMocks
+    private StringChangerService stringChangerService;
+
+    @Mock
+    private StringLowerUpperService stringLowerUpperService;
+
+    @Before
+    public void setUp(){
+        doReturn("startinput").when(stringChangerService).addStart(anyString(), anyString());
+        //doReturn("endinput").when(stringChangerService).addEnd(anyString(), anyString());
```
Comment (Cherkashyn) [inline src/test/java/ua/mateacademy/tania/lesson11/StringChangerServiceMockSpyTest.java:31]: Delete this line ))

File: src/main/java/ua/mateacademy/tania/lesson12/json/MateGroup.java [hunk 2]
```diff
@@ -112,6 +123,12 @@ public static MateGroup mateGroupExampleCreator() {
...
 
+        List<Laptop> laptops = new ArrayList<>();
+        laptops.add(new Laptop("Dell", "Inspiron 15 3567", "USA"));
+        laptops.add(new Laptop("HP", "250 G6", "USA"));
+        mateGroup.setLaptops(laptops);
+
+
         return mateGroup;
     }
 }
```
Comment (Cherkashyn) [inline src/main/java/ua/mateacademy/tania/lesson12/json/MateGroup.java:134]: Did you add List<laptop> laptops to MateGroup constructor?

**Review by Cherkashyn — APPROVED** [review]
Good.

**Review by SerhiiPasko — APPROVED** [review]
Good

**Pull request merged**